src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -96,45 +83,42 @@ public IntakeSubsystem(WPI_TalonFX motorOuterAxle,
     // Methods
 
     /**
-     * Manually sets the speed of the inner and outer axle motors.
+     * Manually sets the speed of the motor
      *
      * @param speed
      */
     public void setSpeed(double speed) {
-        motorInnerAxle.set(speed);
-        motorOuterAxle.set(speed);
+        motor.set(speed);
     }
 
     /**
-     * Spins motors inwards and updates motor state.
+     * Spins motor inwards and updates motor state.
      * Runs if Intake is extended which is when the solenoid is retracted.
      */
     public void intakeIn() {
         if (intakeSolenoidState == RETRACT) {
-            motorOuterAxle.set(INTAKE_IN_SPEED);
-            motorInnerAxle.set(INTAKE_IN_SPEED);
+            motor.set(INTAKE_IN_SPEED);
             intakeMotorState = IN;
         }
     }
 
     /**
-     * Spins motors outwards and updates motor state.
+     * Spins motor outwards and updates motor state.
      * Runs if Intake is extended which is when the solenoid is retracted.
      */
     public void intakeOut() {
         if (intakeSolenoidState == RETRACT) {
-            motorOuterAxle.set(INTAKE_OUT_SPEED);
-            motorInnerAxle.set(INTAKE_OUT_SPEED);
+            motor.set(INTAKE_OUT_SPEED);
             intakeMotorState = OUT;
         }
     }
 
     /**
-     * Stops motors and updates motor state
+     * Stops motor and updates motor state
      */
     public void intakeStop() {
-        motorOuterAxle.stopMotor();
-        motorInnerAxle.stopMotor();
+        motor.set(0);
```

Comment:
Is there a reason you are both stopping the motor and then also telling it to coast to a stop after

Reply:
oops i forgot to remove stopMotor();